serving/docker/pytorch-inf2.Dockerfile

```diff
@@ -13,10 +13,11 @@ FROM ubuntu:20.04
 ARG djl_version=0.23.0~SNAPSHOT
 ARG torch_version=1.13.1
 ARG python_version=3.8
-ARG torch_neuronx_version=1.13.1.1.7.0
-ARG transformers_neuronx_version=0.3.32
-ARG transformers_version=4.28.1
-ARG accelerate_version=0.18.0
+ARG torch_neuronx_version=1.13.1.1.8.0
+ARG transformers_neuronx_version=0.4.60
+ARG neuronx_distributed_version=0.1.0
+ARG transformers_version=4.30.1
+ARG accelerate_version=0.20.3
 ARG diffusers_version=0.14.0
```

Comment:
should we update this as well? or does neuronx need this version specifically

Reply:
neuronx doesn't set a version. I tried to fix it for stablity